src/queries/index.js

```diff
@@ -79,6 +84,7 @@ export const getQuery = (query, filterOptions = {}) => {
     [QUERY_TYPES.WASTE_STREET]: WASTE_STREET,
     [QUERY_TYPES.VOUCHER]: GET_VOUCHER,
     [QUERY_TYPES.VOUCHERS_CATEGORIES]: GET_VOUCHERS_CATEGORIES,
+    [QUERY_TYPES.VOUCHERS_REDEEMED]: GET_REDEEMED_VOUCHERS,
```

Comment:
it would have make sense to change `GET_REDEEMED_VOUCHERS` as well, or what do you say?

Reply:
Yes, it will be more logical with this change.  👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/7b92f2e633df56952015b99c46f92a330551cb21